### PR TITLE
Added European time support for Archer

### PR DESCRIPTION
**Pull request by guykeller**
## Status 
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/18909

## Description
Added European time support for Archer (implemented by @hirshbergshachar ).

## Does it break backward compatibility?
   - No

## Must have
- [ ] Code Review

### CODE REVIEW

File: Integrations/integration-ArcherRSA.yml [hunk 1]
```diff
@@ -2130,6 +2170,19 @@ script:
         return incidents;
     }
 
+    function convertToAmericanTime(lastTime) {
```
Comment (hirshbergshachar) [inline Integrations/integration-ArcherRSA.yml]: Do we use this?

File: Integrations/integration-ArcherRSA.yml [hunk 2]
```diff
@@ -2719,7 +2781,7 @@ script:
             fieldIdToSearchOn = JSON.stringify(nameToFieldMapping[fieldNameToSearchOn].Id);
             dateOperator = "GreaterThan";
             var day = 60 * 1000 * 60 * 24;
-            searchValue = toDateArcherString(new Date(new Date().getTime() - (day * 186)));
+            searchValue = toDateArcherString(new Date(new Date().getTime() - (day * 1)));
```
Comment (hirshbergshachar) [inline Integrations/integration-ArcherRSA.yml]: Should be 30 days

File: Integrations/integration-ArcherRSA.yml [hunk 3]
```diff
@@ -2851,6 +2913,7 @@ script:
             tz = 0;
         }
         var offset = tz ? (60 * 1000 * new Date().getTimezoneOffset()) : 0;
+        offset = tz ? parseInt(tz) : 0;
```
Comment (hirshbergshachar) [inline Integrations/integration-ArcherRSA.yml]: Please parse the offset to a time object and check that we do the addition correctly with lastTime

Comment (guykeller) [reply Integrations/integration-ArcherRSA.yml]: Planned on fixing the time parsing issue in a second unrelated PR, if that's ok with you

File: Integrations/integration-ArcherRSA.yml (hunk 1 above)
Comment (guykeller) [reply Integrations/integration-ArcherRSA.yml]: removed

File: Integrations/integration-ArcherRSA.yml (hunk 2 above)
Comment (guykeller) [reply Integrations/integration-ArcherRSA.yml]: Fixed

**Pull request merged**